__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("should respond with a 200 status code", () => {
```

Comment:
This isn't necessary, you're testing both the response from the server and the status code in one on line 21. With integration testing you're absolutely ok to combine these two things as they are relevant to each other.